src/index.ts

```diff
@@ -27,6 +27,7 @@ export {IAMAuth, RequestMetadata} from './auth/iam';
 export {IdTokenClient, IdTokenProvider} from './auth/idtokenclient';
 export {Claims, JWTAccess} from './auth/jwtaccess';
 export {JWT, JWTOptions} from './auth/jwtclient';
+export {Impersonated, ImpersonatedOptions} from './auth/impersonated';
```

Comment:
You may want to add a test for `Impersonated`:
https://github.com/googleapis/google-auth-library-nodejs/blob/3e0c4c92014f2b7b4543f28ad352f0d135efd08f/test/test.index.ts#L31